bot/resources/tags/under.md

```diff
@@ -0,0 +1,36 @@
+## UseCases of Underscore in Python
```

Comment:
This doesn't work in Discord embeds. It'll show up as the following:
<img width="251" alt="image" src="https://user-images.githubusercontent.com/15021300/206897545-1beb67e8-aa9c-48a3-9998-0c504350bfa3.png">

```suggestion
**UseCases of Underscore in Python**

```